openassessment/xblock/test/test_openassessment.py

```diff
@@ -59,6 +59,33 @@ def test_load_student_view(self, xblock):
         self.assertIsNotNone(grade_response)
         self.assertIn("step--grade", grade_response.body)
 
+    @scenario('data/staff_grade_scenario.xml')
+    def test_load_grade_available_responses_view(self, xblock):
+        """OA XBlock returns some HTML for case if Staff Assessment is configured.
+
+        View basic test for verifying auxiliary view which displays the staff grading area.
+        """
+        xblock_fragment = self.runtime.render(xblock, "grade_available_responses_view")
+        body_html = xblock_fragment.body_html()
+        self.assertIn("GradeAvailableResponsesBlock", body_html)
+        self.assertIn("openassessment__title", body_html)
+        self.assertIn("openassessment__staff-area", body_html)
+        self.assertIn("ui-staff__content", body_html)
+        self.assertNotIn("openassessment__staff-area-unavailable", body_html)
+
+    @scenario('data/basic_scenario.xml')
+    def test_load_grade_available_responses_view_staff_assessment_not_configured(self, xblock):
+        """OA XBlock returns some HTML for case if Staff Assessment is not configured.
+
+        View basic test for verifying auxiliary view which displays the staff grading area.
+        """
+        xblock_fragment = self.runtime.render(xblock, "grade_available_responses_view")
+        body_html = xblock_fragment.body_html()
+        self.assertIn("GradeAvailableResponsesBlock", body_html)
+        self.assertIn("openassessment__title", body_html)
+        self.assertNotIn("ui-staff__content", body_html)
+        self.assertIn("openassessment__staff-area-unavailable", body_html)
```

Comment:
These two tests are identical outside of scenario and the final assert, yes? Let's combine them to avoid duplicating code.

```
@scenario(a)
def test_1(self):
    _staff_assessment_view_helper()
    self.assertNotIn(...)

@scenario(b)
def test_2(self):
    _staff_assessment_view_helper()
    self.assertIn(...)
```